velox/docs/functions/coverage.rst

```diff
@@ -172,14 +172,14 @@ Here is a list of all scalar and aggregate Presto functions with functions that
     binomial_cdf                              from_iso8601_timestamp                    :func:`millisecond`                       st_dimension                              truncate                                      :func:`max`
     bit_count                                 :func:`from_unixtime`                     :func:`minute`                            st_disjoint                               typeof                                        :func:`max_by`
     :func:`bitwise_and`                       from_utf8                                 mod                                       st_distance                               uniqueness_distribution                       merge
-    bitwise_arithmetic_shift_right            geometry_as_geojson                       :func:`month`                             st_endpoint                               :func:`upper`                                 merge_set_digest
-    bitwise_left_shift                        geometry_from_geojson                     multimap_from_entries                     st_envelope                               :func:`url_decode`                            :func:`min`
-    bitwise_logical_shift_right               geometry_invalid_reason                   myanmar_font_encoding                     st_envelopeaspts                          :func:`url_encode`                            :func:`min_by`
+    :func:`bitwise_arithmetic_shift_right`    geometry_as_geojson                       month                                     st_endpoint                               :func:`upper`                                 merge_set_digest
```

Comment:
Thank you for noticing that coverage maps need updating as well. We update these in bulk using the procedure described in https://github.com/facebookincubator/velox/tree/main/velox/functions/prestosql/coverage#readme and submit changes as separate PRs, e.g. #454 . Hence, let's not update these here.